bkg/v2/BKG_v2.0.0-Beta-3.yaml

```diff
@@ -2592,7 +2596,9 @@ components:
           type: string
           maxLength: 30
           description: |
-            Reference number for agreement between shipper and carrier through which optionally includes a certain minimum quantity of cargo that the shipper commits to over a fixed period, and the carrier commits to a certain rate or rate schedule.
+            Reference number for agreement between shipper and carrier, which optionally includes a certain minimum quantity commitment (usually referred as “MQC”) of cargo that the shipper commits to over a fixed period, and the carrier commits to a certain rate or rate schedule.
+            
+            **Condition:** Either a valid `serviceContractReference` or `ontractQuotationReference` must be provided.
```

Comment:
spelling mistake. "ontractQuotationReference" , should have been "contractQuotationReference"